bb/bb.go

```diff
@@ -217,22 +221,25 @@ var config struct {
 	Gosrcroot string
 	Arch      string
 	Goos      string
-	Gopath    string
-	TempDir   string
-	Go        string
-	Debug     bool
-	Fail      bool
+	// GOPATH is several paths, separated by :
+	// We require that the first element be the
+	// basic path that works with u-root.
+	Gopath  string
```

Comment:
I found a few places where it was useful to have a distinguished path that was purely for u-root commands, so I left it this way. I can be convinced to do something better.